.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Recipes added with this pull request (max 5):
```

Comment:
Not sure that we should have a hard limit on this. For some people, dumping everything in one go is the easiest way to get started. Admittedly, we may end up breaking it up in the end, but it would be better for reviewers and contributors to work together to strike a healthy balance for this on a case-by-case basis.